pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/model/patch/DeletePatch.java

```diff
@@ -17,6 +20,7 @@
 @Slf4j
 public class DeletePatch extends ExistingRecordPatch {
     DeleteType deleteType;
+    static ClinicalOrderDao clinicalOrderDao = new ClinicalOrderDao();
```

Comment:
Although `ClinicalOrderDao` appears stateless, it would be safer to make `clinicalOrderDao` an instance variable rather than a static, and then create a new ClinicalOrderDao on-the-fly in `isSafeToDelete()`.

Reply:
Just as you also pointed out below, less and less statics is my goal for today. I will remove static from this and `isSafeToDelete` implementations